src/IB/IBFEMethod.cpp

```diff
@@ -1506,12 +1506,11 @@ void IBFEMethod::endDataRedistribution(Pointer<PatchHierarchy<NDIM> > /*hierarch
             // TODO: d_current_time is actually nan here. Since we don't do
             // any sort of tagging based on the current time I think we can
             // just put in a bogus (nonnegative) value.
-            int finest_level = d_scratch_hierarchy->getFinestLevelNumber();
-            if (finest_level == 0)
+            if (getCoarsestPatchLevelNumber() == 0)
                 d_scratch_gridding_algorithm->makeCoarsestLevel(d_scratch_hierarchy, 0.0 /*d_current_time*/);
             else
                 d_scratch_gridding_algorithm->regridAllFinerLevels(
-                    d_scratch_hierarchy, finest_level - 1, 0.0 /*d_current_time*/, tag_buffer);
+                    d_scratch_hierarchy, getCoarsestPatchLevelNumber() - 1, 0.0 /*d_current_time*/, tag_buffer);
```

Comment:
```suggestion
                    d_scratch_hierarchy, std::max(getCoarsestPatchLevelNumber() - 1, 0), 0.0 /*d_current_time*/, tag_buffer);
```
?

Reply:
See like 1509 :)

Reply:
Oh right.

Reply:
Should there be a subsequent call to `regridAllFinerLevels()` if `getCoarsestPatchLevelNumber()` returns `0`?

Reply:
I think you're right. Let me take a look.